# pos/direction of objects(Monocular Vision)

Header header

# pos/direction of targets detected(num --- number of targets detected)

uint8 target_num
 
bool obstacle_flag
 
uint8[] serial_flag
float32[] target_x
float32[] target_y
float32[] target_img_x
float32[] target_img_y
float32[] target_dir
float32[] target_vx
float32[] target_vy
string[] target_id

